actions/getMumblePost.ts

```diff
@@ -0,0 +1,17 @@
+import { auth } from '@/app/api/auth/[...nextauth]/auth';
+import { IPost, MumblePostService } from '@/services/Mumble/MumblePost';
+import config from '@/config';
+
+export const getMumblePostAction = async (
+  identifier: string,
+  includeReplies: boolean,
+): Promise<IPost> => {
+  const session = await auth();
+  const dataSrc = new MumblePostService(config.mumble.host);
+  return await dataSrc.getPostById({
+    // @ts-ignore
```

Comment:
I create a task for fixing the ts-ignore lines in our codebase: https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/issues/8
In case we don't forget to fix them before final hand-in.

Reply:
Wen need to add a types file on the root of the project to extend the user interface/type